src/components/Tokens/TokenDetails/index.tsx

```diff
@@ -234,13 +238,14 @@ export default function TokenDetails({
 
         <RightPanel onClick={() => isBlockedToken && setOpenTokenSafetyModal(true)}>
           <div style={{ pointerEvents: isBlockedToken ? 'none' : 'auto' }}>
-            <Widget
-              defaultTokens={{
-                [Field.INPUT]: inputToken ?? undefined,
-                default: detailedToken ?? undefined,
+            <Swap
```

Comment:
i forgot about this until today but are we still planning on feature flagging the replacement here?

as for the callback function changes, i would suggest keeping the old ones and using new ones for the `Swap` component :3 and we can delete the old ones once we get rid of feature flag logic